scripts/json-query.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable max-len */
+export default function query(select, from, where, orderBy, rowCount) {
+  // Validate input parameters
+  if (!Array.isArray(select) || !Array.isArray(from) || !Array.isArray(where) || where.length !== 3) {
```

Comment:
can we replace this with !([select, from, where].every(Array.isArray) && where.length === 3)